web-common/src/features/dashboards/time-controls/TimeRangeSelector.svelte

```diff
@@ -238,6 +252,18 @@
         {allTime.label}
       </span>
     </MenuItem>
+    {#if showDefaultItem && $timeControlsStore.defaultTimeRange}
+      <DefaultTimeRangeMenuItem
+        on:before-select={setIntermediateSelection(TimeRangePreset.DEFAULT)}
```

Comment:
The value of intermediate selection should be the humanize value of the ISO duration. If I have the default duration selected in the menu and after that if I try to change to a different preset, I see a value of `Default` being shown in the menu. In some slower systems and networks, this could be an issue.

Reply:
Good catch. Dont think it makes sense to set it to the humanised ISO. The issue was in the value shown for the select label.